src/styles/blocks/body.scss

```diff
@@ -0,0 +1,8 @@
+body {
+  margin: 0;
+
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  color: $main-color;
```

Comment:
The variable `$main-color` is used for the `color` property. Ensure that this variable is defined elsewhere in your SCSS files. If it's not defined, you need to declare it with a valid color value before using it.